compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/pack_and_transpose_level1.mlir

```diff
@@ -2,6 +2,8 @@
 
 // CHECK: #config
 #config = #iree_codegen.lowering_config<tile_sizes = [[8, 8], [4, 4], [0, 0, 4]]>
+// TODO(avarma) - I would like a similar global hoisting of packing_config. Help?
```

Comment:
Can someone please provide me pointers for this?

I tried looking both into `lowering_config` and `affine_map` but am unable to find the infra that gets me the following :-
```
packingConfig = #amdaie.packing_config......
func.func @myfunc(...) ->() {
.
.
%0 = %myOp {packing_config = #packingConfig}
.
.
}
```
I feel it's good to have the above construct.